msteams-platform/m365-apps/prerequisites.md

```diff
@@ -91,18 +91,37 @@ To install Microsoft 365 Beta Channel applications in your test environment:
 
 ### Mobile
 
-Install the Microsoft 365 app to preview your Teams personal tab app running your physical Android device or Android emulator:
+You can preview Teams personal tabs running in Microsoft 365 app for Android by joining the beta program.
+
+Install the Microsoft 365 app to preview your Teams personal tab app running on your physical Android device or Android emulator:
 
 1. Ensure you're using a Google Play [supported Android device](https://support.google.com/googleplay/answer/1727131).
 1. Launch the **Play Store** on your Android device.
-1. Search for "Office" and select **Office (Microsoft 365)**.
+1. Search for your app of choice. 
+    1. For Microsoft 365 app (formerly Office), search office and select **Microsoft 365 app**.
+    1. For Outlook, search outlook and select **Microsoft Outlook**.
```

Comment:
How about:

- search for *office*
- search for *outlook*

Use single quotes or italics for the suggested search terms